index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
```

Comment:
Please update your CSS to stick to figma.
Here you have to change the color
<img width="78" alt="image" src="https://user-images.githubusercontent.com/7243132/170825375-b9274792-13ee-41eb-975e-349d1f376afa.png">

Figma:
<img width="134" alt="image" src="https://user-images.githubusercontent.com/7243132/170825405-5d7a4510-8492-44e3-a50d-c573bca31cbf.png">